website/src/components/IndexPage/getOrganismStatistics.ts

```diff
@@ -68,12 +67,25 @@ const getTotalAndLastUpdatedAt = async (
 const getRecent = async (organism: string, numberDaysAgo: number): Promise<number> => {
     const recentTimestamp = Math.floor(Date.now() / 1000 - numberDaysAgo * 24 * 60 * 60);
     const client = LapisClient.createForOrganism(organism);
-    return (
+    const recentTotal =  (
         await client.call('aggregated', {
             [`${RELEASED_AT_FIELD}From`]: recentTimestamp,
             version: 1,
+            
         })
     )
         .map((x) => x.data[0].count)
         .unwrapOr(0);
-};
+        const recentRevoked =  (
+            await client.call('aggregated', {
+                [`${RELEASED_AT_FIELD}From`]: recentTimestamp,
+                version: 1,
+                [VERSION_STATUS_FIELD]: siloVersionStatuses.revoked
```

Comment:
Not sure this works? As this filters for sequences that are version 1 AND revoked? Which will be 0?

Reply:
I think we need a 2 step process where we get accessions as intermediate step? Unless I misunderstand SILO capabilities @Taepper

Reply:
My understanding is that the `versionStatus` field is always either "LATEST_VERSION" or "REVISED" or "REVOKED". It relates to how the version has been affected by other newer versions.

Reply:
I suspect you are correct.. let me think for a few more minutes

Reply:
So what will happen to older version if the newest version is revoked?
If this is possible:
`PP_01.1` `REVISED`
`PP_01.2` `REVOKED`
I think we would need the intermediate step mentioned by Cornelius with current LAPIS/SILO limitations

Reply:
Sounds good to me! Would you like to create an issue to document the edge cases that still need to be solved? Otherwise, I'm happy with the solution.

Reply:
What does that `versionStatus` field do? I'm confused now.

Reply:
> I'm confused now

Me too 😅

Reply:
(Sorry, when I resolved this I hadn't seen the comments)

Reply:
So here is the code that computes SILO versions https://github.com/loculus-project/loculus/blob/cca00bbc9174e3080e7507802ce838f186babc22/backend/src/main/kotlin/org/loculus/backend/model/ReleasedDataModel.kt#L105

Here is a Claude description:
>First, it checks if the given version is the latest. If so, it returns LATEST_VERSION.
>If not the latest, it checks if there's a revocation version for this accession that's newer than the given version. If so, it returns REVOKED.
>If neither of the above conditions are met, it returns REVISED, indicating that newer versions exist but this version hasn't been revoked.

Thus I think actually there are no edge cases?

Reply:
aah, right, okay, forgot about the exact implementation myself but it makes sense! Then, this should indeed fully cover all cases.

Reply:
(sequences which were submitted, revoked, then reinstated won't be counted as new submissions but that doesn't seem worth an issue)